doc/articles/faq.md

```diff
@@ -234,12 +234,9 @@ as a base for your cross-targeted library.
 
 # Is it possible to make http web requests using the WASM target?
 
-Yes it is possible, but you need to use the provided HttpHandler by Uno.UI.WebAssembly like what we did in the Uno.UI Playground:
+Yes, `HttpClient` works out of the box on .NET 5+ (Uno.Wasm.Bootstrap 2.0.0).
 
-```csharp
-var handler = new Wasm.WasmHttpHandler();
-var httpClient = new HttpClient(handler);
-```
+On older versions, you need to use the HttpHandler provided by Uno.UI.WebAssembly. See [example](https://github.com/unoplatform/Uno.Samples/blob/master/UI/TheCatApiClient/TheCatApiClient/TheCatApiClient.Shared/WebServices/WebApiBase.cs#L18).
```

Comment:
Maybe use a permanent source link here, if the aim is to have guidance for older versions, since conceivably the Cat sample on `master` will be updated to current usage at some point.